app/assets/javascripts/organizations.js

```diff
@@ -0,0 +1,65 @@
+$(function () {
+
+  "use strict";
+
+  $(document).ready(function () {
+    $(".btn.new_account").on("ajax:success", function (e, data) {
+      $(".new_account_form").empty();
+      $(".previous_buttons").addClass("hidden");
+      $(".next_step_wrapper").addClass("hidden");
+      $(".new_account_form").append(data);
+      $("#new_bank_account").on("ajax:success", function (e, data) {
+        if (data["status"] === "error") {
```

Comment:
every time then returns form, I must bind new ajax:success on this form, recursion of a current binding. To my knowledge, it is not able in javascript.